doc/releases/quincy.rst

```diff
@@ -5,6 +5,602 @@ Quincy
+* rgw: fix SignatureDoesNotMatch when extra headers start with 'x-amz' (`pr#53771 <https://github.com/ceph/ceph/pull/53771>`_, rui ma)
+* rgw: fix unwatch crash at radosgw startup (`pr#53761 <https://github.com/ceph/ceph/pull/53761>`_, lichaochao)
+* rgw: handle http options CORS with v4 auth (`pr#53414 <https://github.com/ceph/ceph/pull/53414>`_, Tobias Urdin)
+* rgw: improve buffer list utilization in the chunkupload scenario (`pr#53774 <https://github.com/ceph/ceph/pull/53774>`_, liubingrun)
+* rgw: LDAP fix resource leak with wrong credentials (`pr#50562 <https://github.com/ceph/ceph/pull/50562>`_, Johannes Liebl, Johannes)
+* rgw: optimizations for handling ECANCELED errors from within get_obj_state (`pr#50892 <https://github.com/ceph/ceph/pull/50892>`_, Cory Snyder)
+* rgw: pick http_date in case of http_x_amz_date absence (`pr#53441 <https://github.com/ceph/ceph/pull/53441>`_, Seena Fallah, Mohamed Awnallah)
+* rgw: retry metadata cache notifications with INVALIDATE_OBJ (`pr#52799 <https://github.com/ceph/ceph/pull/52799>`_, Casey Bodley)
+* rgw: rgw_parse_url_bucket() rejects empty bucket names after 'tenant:' (`pr#50625 <https://github.com/ceph/ceph/pull/50625>`_, Casey Bodley)
+* rgw: s3website doesn't prefetch for web_dir() check (`pr#53768 <https://github.com/ceph/ceph/pull/53768>`_, Casey Bodley)
+* rgw: set keys from from master zone on admin api user create (`pr#51601 <https://github.com/ceph/ceph/pull/51601>`_, Ali Maredia)
+* rgw: swift : check for valid key in POST forms (`pr#52739 <https://github.com/ceph/ceph/pull/52739>`_, Abhishek Lekshmanan)
+* rgw: under fips & openssl 3.x allow md5 usage in select rgw ops (`pr#51269 <https://github.com/ceph/ceph/pull/51269>`_, Mark Kogan)
+* rgwlc: prevent lc for one bucket from exceeding time budget (`pr#53561 <https://github.com/ceph/ceph/pull/53561>`_, Matt Benjamin)
+* test/cli-integration/rbd: iSCSI REST API responses aren't pretty-printed anymore (`pr#52283 <https://github.com/ceph/ceph/pull/52283>`_, Ilya Dryomov)
+* test: correct osd pool default size (`pr#51804 <https://github.com/ceph/ceph/pull/51804>`_, Nitzan Mordechai)
+* test: monitor thrasher wait until quorum (`pr#51801 <https://github.com/ceph/ceph/pull/51801>`_, Nitzan Mordechai)
+* tools/ceph-dencoder: Fix incorrect type define for trash_watcher (`pr#51779 <https://github.com/ceph/ceph/pull/51779>`_, Chen Yuanrun)
+* tools/cephfs-data-scan: support for multi-datapool (`pr#50522 <https://github.com/ceph/ceph/pull/50522>`_, Mykola Golub)
+* tools/cephfs: add basic detection/cleanup tool for dentry first damage (`pr#52245 <https://github.com/ceph/ceph/pull/52245>`_, Patrick Donnelly)
+* tools/cephfs: include lost+found in scan_links (`pr#50783 <https://github.com/ceph/ceph/pull/50783>`_, Patrick Donnelly)
+* v17.2.6 (`pr#50895 <https://github.com/ceph/ceph/pull/50895>`_, Ceph Release Team, Casey Bodley, Radosław Zarzyński)
```

Comment:
This was pointed out in the past -- including a tag of the `N-1` release in the release notes for the `N` release is confusing.  If it's a problem with the release notes script itself (as opposed to how it's invoked), please make a note to delete this line manually for now.